libnd4j/flatc-generate.sh

```diff
@@ -1,6 +1,7 @@
+#!/bin/bash
 
 # flatbuffers
-flatc -o ./include/graph/generated -I ./include/graph/scheme -j -b  -t -c ./include/graph/scheme/node.fbs ./include/graph/scheme/graph.fbs ./include/graph/scheme/result.fbs ./include/graph/scheme/config.fbs ./include/graph/scheme/array.fbs ./include/graph/scheme/utils.fbs ./include/graph/scheme/variable.fbs
+flatc  -o ./include/graph/generated -I ./include/graph/scheme -j -b  -t -c  ./include/graph/scheme/node.fbs ./include/graph/scheme/graph.fbs ./include/graph/scheme/result.fbs ./include/graph/scheme/config.fbs ./include/graph/scheme/array.fbs ./include/graph/scheme/utils.fbs ./include/graph/scheme/variable.fbs
```

Comment:
Converted the .txt file here in to a script for generating the right flatbuffers files.